doc/howto/storage_volumes.md

```diff
@@ -107,80 +141,49 @@ Instead of attaching a custom volume to an instance as a disk device, you can al
 
 To do so, you must set the corresponding {ref}`server configuration <server-options-misc>`:
 
-- To use a custom volume to store the backup tarballs:
+- To use a custom volume `my-backups-volume` to store the backup tarballs:
 
-      lxc config set storage.backups_volume <pool_name>/<volume_name>
+      lxc config set storage.backups_volume=my-pool/my-backups-volume
 
-- To use a custom volume to store the image tarballs:
+- To use a custom volume `my-images-volume` to store the image tarballs:
 
-      lxc config set storage.images_volume <pool_name>/<volume_name>
+      lxc config set storage.images_volume=my-pool/my-images-volume
 
 (storage-configure-volume)=
 ## Configure storage volume settings
 
-See the {ref}`storage-drivers` documentation for the available configuration options for each storage driver.
-
-Use the following command to set configuration options for a storage volume:
-
-    lxc storage volume set <pool_name> [<volume_type>/]<volume_name> <key> <value>
-
-The default {ref}`storage volume type <storage-volume-types>` is `custom`, so you can leave out the `<volume_type>/` when configuring a custom storage volume.
+See the {ref}`storage-drivers` documentation for a list of available storage volume configuration options for each driver.
 
-For example, to set the size of your custom storage volume `my-volume` to 1 GiB, use the following command:
+To set the size of custom storage volume `my-volume` to 1 GiB, use the following command:
```

Comment:
```suggestion
To set the size (quota) of custom storage volume `my-volume` to 1 GiB, use the following command:
```

Reply:
Per [Tom's comment](https://github.com/canonical/lxd/pull/14578#issuecomment-2523201351) - indicate that quota is another word for size in LXD docs.

Reply:
I prefer "To set the maximum size of..." if you agree that it communicates the same idea.